MAINTAINERS.yml

```diff
@@ -197,17 +196,16 @@ ARM64 arch:
 ARM Platforms:
   status: odd fixes
   files:
-    - boards/arm/mps*/
-    - soc/arm/arm/
-    - boards/arm/v2m_*/
+    - boards/arm/
+    - soc/arm/
```

Comment:
same as above section

Reply:
fixed, added matching socs for listed boards